test/Interop/Cxx/stdlib/std-module-interface.swift

```diff
@@ -0,0 +1,33 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=std -source-filename=x -enable-cxx-interop -Xcc -stdlib=libc++ -tools-directory=%llvm_obj_root/bin -module-cache-path %t | %FileCheck %s  -check-prefix=CHECK-STD
+// RUN: %target-swift-ide-test -print-module -module-to-print=std.iosfwd -source-filename=x -enable-cxx-interop -Xcc -stdlib=libc++ -tools-directory=%llvm_obj_root/bin -module-cache-path %t | %FileCheck %s  -check-prefix=CHECK-IOSFWD
```

Comment:
Why not have this be `std.string`?

Reply:
No reason, yeah, let's have `std.string` here instead.